src/main/java/com/demo/testingsonar/TestingSonarApplication.java

```diff
@@ -7,6 +7,7 @@
 public class TestingSonarApplication {
 
     public static void main(String[] args) {
+        String s = "this is a test";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "s". ([squid:S1854](https://sonarcloud.io/organizations/davidalonsobadia-github/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.demo%3Atesting-sonar&pullRequest=1&issues=AWPHsWSK4fWm9CZEWTni&open=AWPHsWSK4fWm9CZEWTni)

[ISSUE_KEY]: AWPHsWSK4fWm9CZEWTni